docs/src/guides/working-with-projects.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Working with Projects
+navTitle: Working with Projects
+description: Working with Hub Projects
+order: 12
+group: 2-concepts
+---
+
+## Working with Projects
+
+### In a Component
+
+If you have a component that you want to fetch and display a project using either
+the project's ID or slug
+
+```html
+<arcgis-hub-project-view identifier="dcdev-smith-st-2024" />
+```
+
+In the component, import the `getProject` function from `@esri/hub-common`, and use that
+in the lifecycle hooks to load the project. Directly importing the helper function ensures that the build will produce the smallest payload as compared to importing `HubProjectStore`.
+
+```ts
+import {getProject} from "@esri/hub-common";
+
+async componentWillLoad() {
+  if (!this.project && this.identifier && this.context) {
+    this.project = await getProject(this.identifier, this.context.requestOptions);
+  }
+}
+```
+
+### In an Application
+
+Assuming the following routes in an app...
+
+```js
+//...snip...
+this.route("projects", function () {
+  this.route("project", { path: "/:project_id" }, function () {
+    this.route("edit");
+  });
+  this.route("edit", function () {
+    this.route("new");
+  });
+});
+//...snip...
+```
+
+#### Create a Project
+
+To create a project in the `/projects/edit/new` controller...
+
+```js
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+
+export default class projectsEditNewController extends Controller {
+  @service appSettings;
+
+  @action
+  createProject(project) {
+    const store = HubProjects.create(appSettings.contextManager);
+    return store.create(params.project_id);
+  }
+}
+```
+
+#### Get a Project
+
+In the `/projects/:project_id` route, we get the project by id or slug
+
+```js
+import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
+
+export default class ProjectsProjectRoute extends Route {
+  @service appSettings;
+
+  model(params) {
+    const store = HubProjects.create(appSettings.contextManager);
+    return store.get(params.project_id);
+  }
+}
+```
+
+#### Update a Project
+
+In the `/projects/:project_id/edit` controller we update the project like this
+
+```js
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+
+export default class projectEditController extends Controller {
+  @service appSettings;
+
+  @action
+  updateProject(project) {
+    const store = HubProjects.create(appSettings.contextManager);
+    return store.update(params.project_id);
```

Comment:
should this accept the `project`?

Reply:
clarify what's passed into update

Reply:
yeah - that was a copy-paste typo. Fixed